unfoldingWord/tc-create-app
# Remove check of state since it never exists in sourceRepository

Fixes repo chips not showing

To test:
Open a file. You should see repos chips at top. 
Close file with x in top right by repo name.
No fatals should happen
